label_studio/frontend/src/pages/Organization/index.js

```diff
@@ -0,0 +1,30 @@
+
+import React from 'react';
+import { SidebarMenu } from '../../components/SidebarMenu/SidebarMenu';
+import { PeoplePage } from './PeoplePage/PeoplePage';
+import { WebhookPage } from './WebhookPage/WebhookPage';
+
+const MenuLayout = ({ children, ...routeProps }) => {
+  return (
+    <SidebarMenu
+      menuItems={[
+        PeoplePage,
```

Comment:
When entering Organizations page there's a menu item called "People" that leads to exact same page.

It's better to make `OrganizationPage` a default one. Take a look at how Settings page made, especially how GeneralSettingsPage added to the menu.